### ISSUE TITLE
Alerts should post alerts to an alert buffer

### ISSUE THREAD

**Issue by yaacov** [MohawkTSDB/mohawk#47]
Alerts [1] should post alert changes to an alert buffer [2].

[1] https://github.com/MohawkTSDB/mohawk/tree/master/alerts
[2] https://github.com/ilackarms/message-buffer

a _general_ example for _possible_ post code:

```
client := http.Client{}
...
req, err := http.NewRequest("POST", url, <alerts json as buffer>)
...
resp, err := client.Do(req)
...

```

**Comment by nimrodshn** [comment]
@yaacov should we create our own alert buffer (I was thinking of  a channel..) or use an existing one?

**Comment by yaacov** [comment]
> @yaacov should we create our own alert buffer (I was thinking of a channel..) or use an existing one?

I will add a small alert buffer for the testing #51 :-) but this should work with manageiq, so it must work with scott's alert buffer.

**Comment by yaacov** [comment]
I will take it ...

**Comment by yaacov** [comment]
closed by #54
